terraform/iot_analytics/README.md

```diff
@@ -0,0 +1,94 @@
+| Pub/Sub topic        |         `input`         | The input Pub/Sub topic for the sample pipeline.                                                                                                                                                                                            |
+| Pub/Sub subscription |     `messages-sub`      | The subscription to the `input` topic that is actually used by the Dataflow pipeline.                                                                                                                                                       |
+| Service account      |    `my-dataflow-sa`     | Dataflow worker servive account. It has storage admin, Dataflow worker, metrics writer and Pub/Sub editor roles assigned at project level.                                                                                                  |
+| VPC network          |     `dataflow-net`      | If the project is created from scratch, the default network is removed and this network is re-created with a single regional sub-network.                                                                                                   |
+| Cloud NAT            |     `dataflow-nat`      | Cloud NAT in the region specified by the user, in case the Dataflow workers need to reach the Internet. This is not necessary for the sample pipeline provided.                                                                             |
+| Bigtable Instance    | `clickstream-analytics` | BigTable instance to store the information for enrichment of incoming messages                                                                                                                                                              |
+| BigQuery Dataset     | `clickstream_analytics` | BigQuery dataset where the tables will be created.                                                                                                                                                                                          |
+| BigQuery Table       |       `wikipedia`       | Store the processed records from dataflow job.                                                                                                                                                                                              |
+| BigQuery Table       |      `deadletter`       | Table for all the records that cannot be parsed, it contains more details about the error found processing each record.                                                                                                                     |
+| Firewall rules       |      Several rules      | Ingress and egress rules to remove unnecessary traffic, and to ensure the traffice required by Dataflow. If you want to access a VM using SSH, apply the network tag `ssh` to that instance. Same for `http-server` and for `https-server`. |
+
```

Comment:
Please update this file to make sure it matches this solution guide. The content seems to be mixed with other solution guides.